src/Parser/AfflictionWarlock/Modules/Features/UABuffTracker.js

```diff
@@ -72,7 +72,7 @@ class UABuffTracker extends Module {
   }
 
   suggestions(when) {
-    const unbuffedTicksPercentage = this.unbuffedTicks / this.totalTicks;
+    const unbuffedTicksPercentage = this.unbuffedTicks / (this.totalTicks > 0 ? this.totalTicks : 1);
```

Comment:
Cleaner to do `const unbuffedTicksPercentage = (this.unbuffedTicks / this.totalTicks) || 1;`